tts-evaluation-MCD-DTW.ipynb

```diff
@@ -0,0 +1,495 @@
+{
```

Comment:
How did you get the mel spectrograms from the models?

Reply:
For radTTS I used: https://github.com/NVIDIA/radtts
For fastpitch, I used the method described here: https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/tts_en_fastpitch

Reply:
Do have a code snippet that you can add to the notebook?

Reply:
I can add for fastpitch, for radTTS I had to do some changes in their inference script. That wont be possible in the notebook.